# DO NOT CHANGE UNLESS YOU REALLY KNOW WHAT YOU ARE DOING!
simTimeStep: 0.1
loadedParam: true
